less/home-page.less

```diff
@@ -40,36 +53,63 @@
   }
 
   .walk {
+    width: 100%;
     margin-top: 60px;
     border-top: 1px solid @main-border-color;
     padding: 50px 60px;
     display: flex;
     justify-content: center;
+    @media @mobile {
+      width: 100%;
+      padding: 0;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      #a { order:2}
+      #b { order:1}
+      #c {order: 3}
+      #img {order: -1}
+ 
+    }
 
     .walk-text {
       width: 50%;
```

Comment:
You should add a `@mobile` media query here of `100%` instead of `50%` so it can take up the entire space